code/modules/unit_tests/teleporters.dm

```diff
@@ -0,0 +1,10 @@
+/datum/unit_test/auto_teleporter_linking/Run()
+	// Put down the teleporter machinery
+	var/obj/machinery/teleport/hub/hub = allocate(/obj/machinery/teleport/hub)
+	var/obj/machinery/teleport/station/station = allocate(/obj/machinery/teleport/station, locate(run_loc_floor_bottom_left.x + 1, run_loc_floor_bottom_left.y, run_loc_floor_bottom_left.z))
+	var/obj/machinery/computer/teleporter/computer = allocate(/obj/machinery/computer/teleporter, locate(run_loc_floor_bottom_left.x + 2, run_loc_floor_bottom_left.y, run_loc_floor_bottom_left.z))
```

Comment:
this unit test does not account for placement order

Reply:
does it matter

Reply:
Maybe? If you place them in different orders a regression might cause one to fail. It's just a coverage thing. I suppose it's fine as is.